client/my-sites/marketing/connections/services-group.jsx

```diff
@@ -27,7 +26,7 @@ import './services-group.scss';
 /**
  * Module constants
  */
-const NUMBER_OF_PLACEHOLDERS = config.isEnabled( 'mastodon' ) ? 5 : 4;
+const NUMBER_OF_PLACEHOLDERS = 4;
```

Comment:
Having a variable placeholder count seems overkill. The height of the placeholders and the actual height of the rendered items is quite different, so this is not noticeable anyway.

Reply:
I agree on this point!